akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -74,3 +73,42 @@ and ensure all nodes are in this state
   
 Full documentation about enforcing these checks on joining nodes and optionally adding custom checks can be found in  
 @ref:[Akka Cluster configuration compatibility checks](../cluster-usage.md#configuration-compatibility-check).
+
+
+## Rolling Updates and Migrating Akka
+
+### From Akka 2.5 to Akka 2.6
+
+#### From Java serialization to Jackson
```

Comment:
Do we really need that fourth level of header here? Perhaps we could just skip it and use the L3 header for this section?

Reply:
I agree, we could even end the section with something that a similar approach can be used when changing between other serializers, such as between Jackson and Protobuf.